db/migrate/20221021162236_devise_token_auth_create_users.rb

```diff
@@ -0,0 +1,49 @@
+class DeviseTokenAuthCreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table(:users) do |t|
+      ## Required
+      t.string :provider, :null => false, :default => "email"
+      t.string :uid, :null => false, :default => ""
+
+      ## Database authenticatable
+      t.string :encrypted_password, :null => false, :default => ""
+
+      # ## Recoverable
+      # t.string   :reset_password_token
+      # t.datetime :reset_password_sent_at
+      # t.boolean  :allow_password_change, :default => false
+
+      # ## Rememberable
+      # t.datetime :remember_created_at
+
+      # ## Confirmable
+      # t.string   :confirmation_token
+      # t.datetime :confirmed_at
+      # t.datetime :confirmation_sent_at
+      # t.string   :unconfirmed_email # Only if using reconfirmable
+
+      ## Lockable
+      # t.integer  :failed_attempts, :default => 0, :null => false # Only if lock strategy is :failed_attempts
+      # t.string   :unlock_token # Only if unlock strategy is :email or :both
+      # t.datetime :locked_at
+
+      ## User Info
+      # t.string :name
+      # t.string :nickname
+      # t.string :image
+      t.string :email
+      t.string :gender
```

Comment:
Maybe we can a enum instead of a string, so we limit the possible options on this field, what do you think?

Reply:
I think you can also remove comments in this migration. If later you need to add some extra columns you can create a new migration.